test/atc_test.coffee

```diff
@@ -175,36 +174,19 @@ describe 'hubot-atc', ->
 
   describe "release", ->
     context "with an app", ->
+      beforeEach ->
+        addApplication(@room, "hubot")
+
       context "and an environment", ->
-        it_allows_you_to_release = ->
```

Comment:
With the removal of the lock expiry code, this was only being used once, so I inlined it's contents